examples/react-client/minimal-react/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "minimal-react-example",
+  "name": "@fishjam-dev/minimal-react-example",
```

Comment:
WDYT about using different prefix here, for example `@fishjam-example/minimal-react-example`?

Soon we need to update `fishjam-dev` in package names anyway, so if we use different prefix here, we won't have to update it

Reply:
I've change e2e packages to `@fishjam-e2e` and example apps to `@fishjam-example`